acm/src/main/java/org/literacybridge/acm/gui/assistants/Deployment/SystemPrompts.java

```diff
@@ -0,0 +1,192 @@
+package org.literacybridge.acm.gui.assistants.Deployment;
+
+import org.literacybridge.acm.config.ACMConfiguration;
+import org.literacybridge.acm.store.*;
+
+import static org.literacybridge.acm.Constants.*;
+
+import java.io.File;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SystemPrompts {
+    public final static String SYSTEM_MESSAGE_CATEGORY = "System Messages";
+    private static final MetadataStore store = ACMConfiguration.getInstance()
+            .getCurrentDB()
+            .getMetadataStore();
+
+    private final String title;
+    private String language;
+    private String description;
+    private String categoryId;
+    private File shortPromptFile;
```

Comment:
I decided to check for duplicates in the store so I could rename them to human readable form. Other than that I do not use the store for system prompts.

Reply:
I'm confused here. When we say "store", we mean the ACM content database. This project is all about putting all the prompts, "system" and "playlist" into the store/content database, instead of just a known location in the file system.